### PR TITLE
Add information about the MB_JETTY_PORT variable

### PR DESCRIPTION
**Pull request by jgoux**

### CODE REVIEW

**Comment by Stayallive** [comment]
Maybe also add documentation about the `MB_JETTY_HOST` one, that sets the host the jetty server listens on.

**Comment by agilliland** [comment]
I'll add a note about the `MB_JETTY_HOST` in a separate PR.  making a note now to just flush out a little bit about all available env vars supported by the app.

File: docs/operations-guide/running-the-metabase-jar-file.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ It's that simple.  This will start the Metabase application using all of the def
 
 At this point your ready to go!  You can access your new Metabase server on port 3000, most likely at [localhost:3000](http://localhost:3000)
 
+You can use another port than 3000 by setting the `MB_JETTY_PORT` environment variable before running the jar
```
Comment (bellisman) [inline docs/operations-guide/running-the-metabase-jar-file.md]: How do you do that?

Comment (chrisgeek) [reply docs/operations-guide/running-the-metabase-jar-file.md]: How is MB_JETTY_PORT variable added please. am running windows.

Comment (jrezalla) [reply docs/operations-guide/running-the-metabase-jar-file.md]: In windows environment use SET MB_JETTY_PORT=12345

**Pull request merged**